app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -65,6 +65,7 @@ def reload_answer
 
   # Publish all the graded submissions.
   def publish_all
+    authorize!(:publish_all, @assessment)
```

Comment:
`can?(:publish_all, Course::Assessment::Submission.new(assessment: @assessment))`, I think we don't want tutors to publish all submissions (?)